### PR TITLE
T15702 documentation

### PR DESCRIPTION
**Pull request by pwithnall**
https://phabricator.endlessm.com/T15702

### CODE REVIEW

File: src/docs/eos-autoupdater.8 [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+.PP
+\fBeos\-autoupdater\fP is designed to be run by \fBsystemd\fP(1) on a timer
+(the \fBeos\-autoupdater.timer\fP unit) which is set to poll for updates around
+once an hour. \fBeos\-autoupdater\fP should not typically be run manually, but
+can be triggered by starting the \fBeos\-autoupdater.service\fP unit. This will
+trigger a poll for updates (and it potentially fetches and applies any which
+are found). See \fBsystemctl\fP(1).
+.PP
+To interact with \fBeos\-updater\fP more directly, use the
+\fBeos\-autoupdater\-ctl\fP(8) tool.
```
Comment (cosimoc) [inline src/docs/eos-autoupdater.8]: This should be `eos-updater-ctl`

File: data/docs/eos-update-server.conf.5 [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+updates to other computers on the local network.
+.PP
+The configuration file contains a single section, \fI[Local Network Updates]\fP,
+whose keys are described below.
+.PP
+Default values are stored in
+\fI/usr/share/eos\-updater/eos\-update\-server.conf\fP, which must always
+exist. To override the configuration, copy it to
+\fI/usr/local/share/eos\-updater/\fP or \fI/etc/eos\-updater/\fP. Configuration
+in \fI/etc\fP overrides that in \fP/usr\fP.
```
Comment (cosimoc) [inline data/docs/eos-update-server.conf.5]: Should it not be `\fI/usr\fP`?

File: data/docs/eos-updater.conf.5 [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+provides the necessary configuration for sources which need it.
+.PP
+The configuration file contains a mandatory section, \fI[Download]\fP, and an
+optional per\-source section, \fI[Source "volume"]\fP, whose keys are described
+below.
+.PP
+Default values are stored in \fI/usr/share/eos\-updater/eos\-updater.conf\fP,
+which must always exist. To override the configuration, copy it to
+\fI/usr/local/share/eos\-updater/\fP or \fI/etc/eos\-updater/\fP. Configuration
+in \fI/etc\fP overrides that in \fP/usr\fP.
```
Comment (cosimoc) [inline data/docs/eos-updater.conf.5]: Same as above

**Comment by pwithnall** [comment]
Review comments fixed. Thanks for the detailed review!

**Comment by pwithnall** [comment]
And I’ve just pushed two additional commits to update the README and TODO files, which finishes off this task.

**Pull request merged**